domain/src/main/java/edu/nextstep/camp/calculator/domain/validator/CalculatorInputValidator.kt

```diff
@@ -0,0 +1,24 @@
+package edu.nextstep.camp.calculator.domain
+
+class CalculatorInputValidator {
+    companion object {
+        fun isValidOperationMark(text: String): Boolean {
+            return if (CalculatorInputUtil.isOperationMarkRegex(text)) {
+                true
+            } else {
+                throw IllegalArgumentException("The Mark is not operation mark use: [× - + ÷]")
+                false
```

Comment:
```suggestion
                throw IllegalArgumentException("The Mark is not operation mark use: [× - + ÷]")
                false
```
throw를 던지게 되면 throw 아래 로직은 동작되지 않아요.
즉, 이 함수는 false를 절대로 리턴할 수 없는 구조에요.
의도하신 것처럼 throw를 던지시거나 false를 반환하시거나 둘 중 하나만 하는 것은 어떨까요?

Reply:
return false를 삭제하고 싶은데,
혹시
if () true 
else throw 
가 괜찮은 코드인가요?

Reply:
말씀하신대로 작성하셔도 괜찮습니다.
하지만 여기에서 좀 더 나아가서, isValidOperationMark 함수가 불필요해보여요.
isValidOperationMark 함수를 호출하는 대신, 외부에서 CalculatorInputUtil.isOperationMarkRegex(text) 를 호출해주는 것이 좋을 것 같아요.